wordpress-mobile/WordPress-iOS
# Update to AFNetworking 3.1

Fixes #5640 

Updates AFNetworking to version 3.1

To test:
 - Login in the app with  a WP.com account
 - Have a general look on all sections of a WP.com site
 - Test create a post with a media object (image or video) and see if the upload process goes correctly
 - Have a look on Reader
 - Check the gravatar set image section

Needs review: @jleandroperez do you mind have a look on this? specially the gravatar image reset cache code.
